logging/es/odfe/bin/run_securityadmin.sh

```diff
@@ -27,3 +27,6 @@ while [ $rc != 0 ] && [ $counter -lt 3 ]; do
   echo "RC: $rc"
   ((counter++))
 done
+
+echo "Exit message: $rc"
+exit $rc
```

Comment:
nit: add a newline

Reply:
I went ahead and added the new line at the bottom (though it's a bit hard to tell from this view).